libs/vkd3d/command.c

```diff
@@ -3678,6 +3679,19 @@ static void STDMETHODCALLTYPE d3d12_command_list_DrawIndexedInstanced(d3d12_comm
             iface, index_count_per_instance, instance_count, start_vertex_location,
             base_vertex_location, start_instance_location);
 
+    if (!list->has_valid_index_buffer)
+    {
+        FIXME_ONCE("Application attempts to perform an indexed draw call without index buffer bound.\n");
+        /* We are supposed to render all 0 indices here. However, there are several problems with emulating this approach.
+         * There is no robustness support for index buffers, and if we render all 0 indices,
+         * it is extremely unlikely that this would create a meaningful side effect.
```

Comment:
gl_VertexIndex to draw quads/tris?

Reply:
gl_VertexIndex is always going to return 0 since the indices are 0.